editors/code/src/util.ts

```diff
@@ -0,0 +1,51 @@
+import * as vscode from 'vscode';
+import { inspect } from "util";
+
+// adapted from https://github.com/rust-analyzer/rust-analyzer/blob/ab432b36de5d6a370dbaad923f9a475a00fbf220/editors/code/src/util.ts#L16
```

Comment:
Very nice steal! The default `console.log` doesn't tell you the source of the log apart from the base name :)